src/app/output/output.component.spec.ts

```diff
@@ -22,4 +27,15 @@ describe('OutputComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should update output string based on text in output textbox', async () => {
```

Comment:
This test is unnecessary as it does not test the true intended behavior of this component.  Please remove and once the true behavior is implemented, add the appropriate tests then.

Reply:
We still want to be able to edit whatever is in the output box. The user may make a small change (like remove the obabel header or change the identity of an atom) then copy-paste it into their next input.

Reply:
okay got it. Can you update to use harnesses here?

Reply:
Yes.